plugins/dell-dock/fu-dell-dock-common.h

```diff
@@ -39,9 +39,22 @@
 #define ATOMIC_HUB1_PID			    0x541A
 #define DELL_VID			    0x413C
 
+/* device IDs: ec*/
+#define DELL_DOCK_HID_VID 0x413C
+#define DELL_DOCK_HID_PID 0xB06E
+
 #define DOCK_BASE_TYPE_UNKNOWN 0x0
 #define DOCK_BASE_TYPE_SALOMON 0x04
 #define DOCK_BASE_TYPE_ATOMIC  0x05
 
+typedef struct {
+	guint8 dock_type;
+	guint16 vid;
+	guint16 pid;
+	const gchar *instance_id;
+} DOCK_HWDEV_INFO;
```

Comment:
Can we pick a better name?  I'm thinking something like `DellDockComponent`

Reply:
fixed in 271ccab.